src/main/java/seedu/eylah/expensesplitter/logic/commands/PaidCommand.java

```diff
@@ -38,38 +38,53 @@ public PaidCommand (Index indexOfPersonPaid, String amountPaid) {
         requireNonNull(indexOfPersonPaid);
         requireNonNull(amountPaid);
         this.indexOfPersonPaid = indexOfPersonPaid;
-        this.amountPaid = amountPaid;
+        if (amountPaid.equals("")) {
+            this.amountPaid = null;
```

Comment:
For this, I'm not sure but might be safer to set amount paid as 0 for cases where the input string is "" to prevent NullPointerException or like when this amountPaid is added elsewhere. (Then null will be added, I'm not sure if this will throw an exception)

Reply:
yup you are right my bad! Understood, ill focus on refining this details later. For now ill focus on getting a working release out before tmr